docs/partials/_cloud_free_plan.mdx

```diff
@@ -1,4 +1,4 @@
 To get started with Cypress Cloud,
 [sign up](https://cloud.cypress.io/signup) for our free plan.
 
-Free plans allow **3** users and **500** monthly test results, while exceeding this limit requires upgrading the plan. Please see our [pricing page](https://www.cypress.io/pricing) for more details.
+Free plans allow **50** users and **500** monthly test results, while exceeding this limit requires upgrading the plan. Please see our [pricing page](https://www.cypress.io/pricing) for more details.
```

Comment:
Cool. Just confirming there are no Team and Business plans references that need any editing while we're in here?

Reply:
Correct, I don't see anything for Team or Business